golang/go-cloud-run-hello-world/.vscode/tasks.json

```diff
@@ -0,0 +1,42 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "presentation": {
+        "panel": "shared",
+        "showReuseMessage": true,
+    },
+    "tasks": [
+        {
+            "group": "build",
+            "label": "Build container image using local Docker",
+            "type": "process",
+            "command": "skaffold",
+            "args": [
+                "build",
+                "-p=no-push"
+            ],
+            "problemMatcher": [],
+        },
+        {
+            "label": "Deploy to Kubernetes cluster",
+            "type": "process",
+            "command": "skaffold",
+            "args": [
+                "run"
+            ],
+            "problemMatcher": [],
+        },
+        {
+            "label": "View application logs on Kubernetes",
+            "type": "process",
+            "command": "kubectl",
+            "args": [
+                "logs",
+                "--selector",
+                "app=hello-world"
+            ],
+            "problemMatcher": [],
+        }
+    ]
+}
```

Comment:
I'm not sure what to do with this file, especially given the discussion to leave skaffold out of the Cloud Run samples.